packages/button/src/Button.tsx

```diff
@@ -6,12 +6,71 @@
  *
  */
 
-import React from 'react';
-import PropTypes from 'prop-types';
+import React, { ButtonHTMLAttributes } from 'react';
 import styled from '@emotion/styled';
 import { css } from '@emotion/core';
 import { colors, spacing, spacingUnit, misc, fonts, animations, breakpoints, mq } from '@ndla/core';
 
+type Appearance =
+  | 'outline'
+  | 'link'
+  | 'stripped'
+  | 'lighter'
+  | 'inverted'
+  | 'invertedOutline'
+  | 'ghostPill'
+  | 'ghostPillInverted'
+  | 'ghostPillOutline'
+  | 'ghostPillOutlineInverted'
+  | 'large'
+  | 'clippedButton'
+  | 'clippedButtonOutline'
+  | 'clippedButtonAttachment'
+  | 'clippedButtonAttachmentOutline'
+  | 'clippedButtonLarge'
+  | 'clippedButtonOutlineLarge'
+  | 'clippedButtonAttachmentLarge'
+  | 'clippedButtonAttachmentOutlineLarge';
+
+type Size = 'xsmall' | 'small' | 'normal' | 'medium' | 'large';
+type Border = 'normal' | 'rounded' | 'sharpened';
+type Width = 'auto' | 'full';
+type TextAlign = 'center' | 'left' | 'right';
+type ButtonType = 'button' | 'submit' | 'reset';
+
+interface Props extends Partial<ButtonHTMLAttributes<HTMLButtonElement>> {
```

Comment:
Burde disse eksportere for at bruk av Button andre plasser skal valideres?

Reply:
Var det et så dumt spørsmål at du ikkje gidder å svare?

Reply:
Overså kommentaren, sorry. Det høres ut som en god ide ja!